doc/pfc_asym/PFC_Asymmetric_Test_HLD.md

```diff
@@ -428,3 +429,27 @@ setup, pfc_storm_runner, enable_pfc_asym
 
 - Teardown:
   - Stop ARP responder
+
+### Test case # 5 - Priority Group dropped packets testing
+#### Test objective
+Asymmetric PFC is disabled. Verify ingress Priority Group drop packets counter increasing
```

Comment:
How ingress PG drops are related to assymetric PFC feature? It should be part of drop counters test (but being able to run only on RPC image since using shaper settings via sai thrift)